cobbler/services.py

```diff
@@ -58,14 +55,14 @@ def __xmlrpc_setup(self):
         if self.remote is None:
             self.remote = xmlrpc.client.Server(self.server, allow_none=True)
 
-    def settings(self) -> Settings:
+    def settings(self, **kwargs):
```

Comment:
Why `kwargs` was appended here?

Reply:
The reason is that we append arguments like that we start appending arguments [here](https://github.com/cobbler/cobbler/blob/8899706930eb8d0475314a5d62ab49edd2c5a353/svc/services.py#L52) and thus we always hand over a dictionary with an unknown number of arguments. If this would be missing we would get keyword errors.

Call to the function can be seen in the following.: https://github.com/cobbler/cobbler/blob/8899706930eb8d0475314a5d62ab49edd2c5a353/svc/services.py#L105